# Type: docs
# Subtype: D3
# English Summary: Added license information to the header of the file
# Chinese Summary: 补充了文件头部的许可证信息
# Entry Number: 19

# === Git Diff Content ===

diff --git a/pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ControllerMetrics.java b/pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ControllerMetrics.java
index fc9c1a7ba..fb41b84e9 100644
--- a/pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ControllerMetrics.java
+++ b/pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ControllerMetrics.java
@@ -1,3 +1,18 @@
+/**
+ * Copyright (C) 2014-2015 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.linkedin.pinot.common.metrics;
 
 import com.yammer.metrics.core.MetricsRegistry;
